### PR TITLE
White space

### PR DESCRIPTION
**Pull request by jklwn**
This tries to make indentation consistent by converting leading spaces to tabs in the compiler and RTL sources. Comments not beginning with two single quotes ('') in .bas and .bi files are removed just as comments in .c and .h files not enclosed by "/* .... */".

JK

### CODE REVIEW

File: inc/ustring.bi [hunk 1]
```diff
@@ -0,0 +1,681 @@
+
```
Comment (jayrm) [inline inc/ustring.bi]: If this PR is for whitespace, then I do not expect to see a new file inc/ustring.bi added.  ustring.bi is also added in PR #155 and the 2 versions are different.  Maybe this was added by mistake?

File: src/compiler/ast-node-bop.bas [hunk 2]
```diff
@@ -1539,7 +1539,7 @@ function astNewBOP _
 			return rtlMathLongintMOD( dtype, l, ldtype, r, rdtype )
 		end select
 
-	' Trap ANDALSO, ORELSE, handle floats, and convert to IIF
```
Comment (jayrm) [inline src/compiler/ast-node-bop.bas]: Here, and in many other files, comments with single quote are stripped.  My preference is 2 single quotes to mark a comment, but that does not mean single quoted comments should be removed.

File: src/compiler/stabs.bi [hunk 3]
```diff
@@ -1,40 +1,40 @@
-const STAB_TYPE_GSYM 	= 32 'Global symbol
```
Comment (jayrm) [inline src/compiler/stabs.bi]: Comments are removed here and should not be.  Should just leave it as is.

File: tests/ustring/asc.bas [hunk 4]
```diff
@@ -0,0 +1,144 @@
+#include "fbcunit.bi"
```
Comment (jayrm) [inline tests/ustring/asc.bas]: new files in tests/ustring/* should not be part of a whitespace only commit

File: src/rtlib/unix/sys_dylib.c [hunk 5]
```diff
@@ -17,11 +17,11 @@ FBCALL void *fb_DylibLoad( FBSTRING *library )
 							  "./lib%s.so",
 							  NULL };
 
-	// Just in case the shared lib is an FB one, temporarily reset the
```
Comment (jayrm) [inline src/rtlib/unix/sys_dylib.c]: Newer C compilers should accept `// comments' this.  For portability, all C compilers will accept `/* comments */'.  Regardless, the comments should not be removed.

**Comment by jayrm** [comment]
As mentioned before, in an editor that supports real tabs, spacing=4, the whitespace issue is quite irrelevant.  But if you really feel that the whitespace and formatting of fbc source must be addressed, then consider publishing the filter/sanitizer.  There's really no point in addressing whitespace within the code base without a reliable automatic method (program) to apply the rules.

**Comment by jklwn** [comment]
<<<But if you really feel that the whitespace and formatting of fbc source must be addressed, then consider publishing the filter/sanitizer. 

Maybe i will do that.

**Pull request closed without merging**